src/Microsoft.CodeAnalysis.Testing/Microsoft.CodeAnalysis.Analyzer.Testing/AnalyzerTest`1.cs

```diff
@@ -281,20 +282,24 @@ protected async Task<ImmutableArray<Diagnostic>> VerifySourceGeneratorAsync(Solu
                 return ImmutableArray<Diagnostic>.Empty;
             }
 
-            return await VerifySourceGeneratorAsync(Language, sourceGenerators, testState, verifier.PushContext("Source generator application"), cancellationToken);
```

Comment:
📝 Looks like I forgot a `ConfigureAwait(false)` here.

Reply:
💭 Actually it looks like I forgot more than one. Just leave it for now and I'll review the situation later. Most of the time in Roslyn test code we omit them, but I need to determine the best plan for this library.